app/controllers/projects/workflow_executions_controller.rb

```diff
@@ -73,6 +73,11 @@ def context_crumbs
         }]
     end
 
+    def destroy_paths
```

Comment:
Can you add a comment for this method here, in the concern, and the `workflow_executions_controller.rb` file for what these paths are used for (since one of the paths in the index path)